packages/@react-aria/combobox/src/useComboBox.ts

```diff
@@ -27,7 +27,7 @@ import {useMenuTrigger} from '@react-aria/menu';
 import {useMessageFormatter} from '@react-aria/i18n';
 import {useTextField} from '@react-aria/textfield';
 
-interface AriaComboBoxProps<T> extends ComboBoxProps<T> {
+interface ComboBoxAriaProps<T> extends AriaComboBoxProps<T> {
```

Comment:
this interface should probably move to the types as noted in the above comment
then you won't need to do this renaming business